Sources/App/Core/Score.swift

```diff
@@ -62,12 +70,18 @@ enum Score {
             let defaultVersion = versions.latest(for: .defaultBranch),
             let repo = package.repository
         else { return 0 }
+
+        // The number of dependencies is defaulted to a high number so that in the (very) rare case that
+        // we don't have resolved dependency information, package scores are not boosted.
+        let numberOfDependencies = defaultVersion.resolvedDependencies?.count ?? 1_000
```

Comment:
I'm not super happy with this. Can you think of a better way @finestructure?

Reply:
I've opened #1395 with a suggestion.